proto/stat.proto

```diff
@@ -69,16 +69,17 @@ message PlacePointStatAll {
 
 message SourceSeries {
   map<string, double> val = 1;
-  string measurement_method = 2;
-  string observation_period = 3;
-  string import_name = 4;
-  string provenance_domain = 5;
-  string unit = 6;
+  string measurement_method = 3;
+  string observation_period = 4;
+  string import_name = 5;
+  string provenance_domain = 6;
   string scaling_factor = 7;
-  bool is_dc_aggregate = 8;
-  string provenance_url = 9;
+  string unit = 8;
+  bool is_dc_aggregate = 9;
+  string provenance_url = 10;
   // Only used for latest date obs collection.
   map<string, string> place_to_latest_date = 11;
+  reserved 2;
```

Comment:
This message is the same as the old one, except the field number changes?

Reply:
So no need to change this message?

Reply:
This is to match the proto in g3 which is saved in the BT proto now. They need to be exactly the same..

Reply:
should we add an IMPORTANT note in these proto files (and g3) that now we should keep the tag numbers consistent across the two places?

Reply:
Done